# Gait Info
std_msgs/Header header
string event
string side